src/test/java/com/puppycrawl/tools/checkstyle/BaseFileSetCheckTestSupport.java

```diff
@@ -21,7 +21,12 @@
 
 import com.puppycrawl.tools.checkstyle.api.Configuration;
 
-public class BaseFileSetCheckTestSupport extends BaseCheckTestSupport {
+public class BaseFileSetCheckTestSupport extends AbstractModuleTestSupport {
```

Comment:
@romani This just shows our naming scheme is off.
`AbstractModuleTestSupport` is defined as module and means it should handle anything related to checks, filesets, filters, etc.
Because we still need `BaseFileSetCheckTestSupport`, this shows that the `ModuleTestSupport` is actually a `CheckTestSupport` and it doesn't support parents of the Check.
We need to find a solution to this problem